data/json/recipes/weapon/mods.json

```diff
@@ -23,6 +23,19 @@
     ]
   },
   {
+    "result": "crafted_suppressor",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "mechanics",
+    "difficulty": 1,
+    "skills_required": [ "gun", 1 ],
+    "time": 650,
+    "autolearn": true,
+    "qualities": [ { "id": "SAW_M_FINE", "level": 1 } ],
+    "components": [ [ [ "filter_liquid", 1 ], [ "filter_liquid_makeshift", 1 ] ] ]
```

Comment:
Just found that now. I'm going to rework `loudness` somewhat and come back with some figures for review.